packages/Modal/src/components/Header/Header.styles.js

```diff
@@ -1,9 +1,21 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { spacer } from "@paprika/stylers/lib/helpers";
+import stylers from "@paprika/stylers";
+import Heading from "@paprika/heading";
 
 export const Header = styled.div`
   align-items: center;
   display: flex;
   justify-content: space-between;
   padding: 0 ${spacer(2)};
 `;
+
+export const HeaderHeading = styled(Heading)(
+  ({ isSingleLine, hasCloseButton }) => css`
+    margin-right: ${spacer(2)};
+    ${isSingleLine && stylers.truncateText};
+    ${isSingleLine && `padding: ${spacer(2)} 0;`};
+    ${isSingleLine && hasCloseButton && `margin: 0 ${spacer(2)} 0 0; width: 95%;`};
+    ${isSingleLine && !hasCloseButton && `margin: 0;`};
```

Comment:
Anyone know a better way to write this, eg 
```
if(isSingleLine){
stylers.truncateText
padding: ${spacer(2)} 0;
closeButton ? margin: 0 ${spacer(2)} 0 0; width: 95% : margin: 0;
}
```
I couldn't get the syntax to work

Reply:
```javascript
export const HeaderHeading = styled(Heading)(
  ({ isSingleLine, hasCloseButton }) => css`
    margin-right: ${spacer(2)};

    ${isSingleLine &&
      `
      ${stylers.truncateText};
      padding: ${spacer(2)} 0;
      
      ${hasCloseButton ? `margin: 0 ${spacer(2)} 0 0; width: 95%;` : `margin: 0;`}
    `}
  `
);
```

Reply:
thanks @jamiek-acl